packages/trpc/server/routers/viewer/organizations/createTeams.handler.ts

```diff
@@ -51,11 +77,16 @@ export const createTeamsHandler = async ({ ctx, input }: CreateTeamsOptions) =>
     .flatMap((ts) => ts.slug ?? [])
     .concat(userSlugs.flatMap((us) => us.username ?? []));
 
-  const duplicatedSlugs = existingSlugs.filter((slug) => teamNames.includes(slug));
+  const duplicatedSlugs = existingSlugs.filter((slug) =>
+    teamNames.map((item) => slugify(item)).includes(slug)
+  );
 
+  if (duplicatedSlugs.length === teamNames.length) {
+    return { duplicatedSlugs };
+  }
```

Comment:
When input teamNames has the same length that deplicatedSlugs that means every input it's duplicated so we should return duplicatedSlugs w/o creating any team.